src/monitor/src/main/java/ee/ria/xroad/monitor/MetricsProviderActor.java

```diff
@@ -107,11 +149,23 @@ public void onReceive(Object o) throws Exception {
         }
     }
 
+    private boolean isProcessPackageOrCertificateMetric(String name) {
+        return Lists.newArrayList(SystemMetricNames.PROCESSES,
+                SystemMetricNames.PROCESS_STRINGS,
+                SystemMetricNames.XROAD_PROCESSES,
+                SystemMetricNames.XROAD_PROCESS_STRINGS,
+                SystemMetricNames.PACKAGES,
+                SystemMetricNames.PACKAGE_STRINGS,
+                SystemMetricNames.CERTIFICATES,
+                SystemMetricNames.CERTIFICATES_STRINGS
+        ).contains(name);
```

Comment:
Instead of creating new list every time it could be statically defined list

Reply:
Corrected.